### PR TITLE
Cut 0.21.0

### PR DESCRIPTION
**Pull request by brancz**
@mxinden

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,14 @@
 
 * [CHANGE] Deprecate specification of Prometheus rules via ConfigMaps in favor of `PrometheusRule` CRDs
```
Comment (mxinden) [inline CHANGELOG.md]: Shall we follow up with this on the release 0.22.0?

Comment (brancz) [reply CHANGELOG.md]: Yeah I felt it was a little too early as the v0.20.0 release was just 3 weeks ago.

Comment (mxinden) [reply CHANGELOG.md]: True that.

**Comment by mxinden** [comment]
This looks good to me.

**Pull request merged**